odd-platform-ui/src/components/Management/DataSourcesList/DataSourcesList.tsx

```diff
@@ -18,6 +18,7 @@ import AddIcon from 'components/shared/Icons/AddIcon';
 import SearchIcon from 'components/shared/Icons/SearchIcon';
 import CancelIcon from 'components/shared/Icons/CancelIcon';
 import NumberFormatted from 'components/shared/NumberFormatted/NumberFormatted';
+import DataSourceItemSkeleton from 'components/Management/DataSourcesList/DataSourceItemSkeleton/DataSourceItemSkeleton';
```

Comment:
I think it would be better to use relative path here. Also this component is a placeholder for the whole list, not just one item - DataSourcesListSkeleton